paths_cli/compiling/topology.py

```diff
@@ -0,0 +1,49 @@
+import os
+from .errors import InputError
+from paths_cli.compiling.root_compiler import compiler_for
+
+def get_topology_from_engine(dct):
+    """If given the name of an engine, use that engine's topology"""
+    # from paths_cli.compiling.engines import engine_compiler
+    engine_compiler = compiler_for('engine')
+    if dct in engine_compiler.named_objs:
+        engine = engine_compiler.named_objs[dct]
+        try:
+            return engine.topology
+        except AttributeError:  # no-cov
+            # how could this happen? passing is correct, to raise the
+            # InputError from MultiStrategyBuilder, but how to test?
+            pass
```

Comment:
Looking at the `MultiMoveStrategyBuilder` the error should be raised if the return is `None`, so this can be rewritten as:
```suggestion
        return getattr(engine, 'topology', None)
```